client/src/App.js

```diff
@@ -1,21 +1,25 @@
 import React, { Component } from "react";
-import Navbar from "./components/Navbar"
+import ReactDOM from "react-dom";
```

Comment:
What is this being used for?

Reply:
I think that's incase we have reactDOM render